Automattic/jetpack
# Connection: lock tokens to prevent IDC during AIOWPM export

## Proposed changes:
* Activate Token Lock for [AIOWPM](https://wordpress.org/plugins/all-in-one-wp-migration/) export to prevent IDC's

### Other information:


## Jetpack product discussion
p9dueE-4Ly-p2
1204904295387809-as-1204812389918149

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
_Easier to test in JN, you'll need two sites._

1. Switch both sites to the branch, connect Jetpack, install All-in-One-WP-Migration plugin.
2. Run DB export in the AIOWPM plugin
3. Open the exported file in a text editor, find `jetpack_private_options` and confirm that `token_lock` is in there.
4. Import the file on the second site using AIOWPM. You'll probably need to re-login into wp-admin, use the password from the first site (import file origin) as the user has been overwritten. If Jetpack got deactivated, reactivate it.
5. Confirm that Jetpack on the second site got disconnected, and the one on original site is still connected.
